Client/Frontend/Browser/TabToolbar.swift

```diff
@@ -71,7 +71,7 @@ open class TabToolbarHelper: NSObject {
         toolbar.addTabButton.addTarget(self, action: #selector(didClickAddTab), for: UIControlEvents.touchUpInside)
         toolbar.addTabButton.addGestureRecognizer(UILongPressGestureRecognizer(target: self, action: #selector(didLongPressAddTab(_:))))
 
-        setTheme(theme: .Normal, forButtons: toolbar.actionButtons)
+        setTheme(theme: .regular, forButtons: toolbar.actionButtons)
```

Comment:
Why the change from "Normal" to "regular"?

Reply:
Changed to regular as the theme is based upon regular browsing so I thought it made sense for consistency, i.e. in the future we may have `regular`, `private` and `tor`